README.md

```diff
@@ -8360,22 +8388,29 @@ class Editor:
         return EditorMemento(self._content)
 
     def restore(self, memento):
-        self.content = memento.getContent()
+        self._content = memento.getContent()
 
 
-----------------------------
+# ----------------------------
 
 editor = Editor()
-editor.type('This is the first sentence')
-editor.type('This is the second.')
+editor.write('This is the first sentence')
+editor.write('This is the second.')
 
 saved = editor.save()
-editor.type('And this is the third')
+editor.write('And this is the third')
 
 print(editor.getContent())  ## This is the first sentence. This is second. And this is third.
 
 editor.restore(saved)
-print(editor.getContent())  ## This is the first sentence. This is second.
+print(editor.getContent())  ## This is the first sentence. This is second
+
+'''
```

Comment:
P.S. I fixed a small bug in line 8391 that corrected the output. 
Additionally, since we are printing the output on each line, this output is no longer needed here.

Reply:
<img width="630" alt="Screenshot 2024-09-25 at 1 10 49 PM" src="https://github.com/user-attachments/assets/9f444db9-3612-4069-9e40-b2c51df51fa8">
I was referring to the section where we have the output as comments